tests/test_core.py

```diff
@@ -1762,7 +1761,6 @@ def test_bigswitch(self):
 
   @no_emterpreter
   def test_biggerswitch(self):
```

Comment:
I mean compile-to-wasm time, so independent of the VM. It actually seems a lot faster with asm2wasm (more so than other things even). Maybe it's hitting some kind of pathological or superlinear case in the wasm backend. Although even the clang->bitcode part is really slow, so, dunno.